examples/hello-world/step-by-step/README.md

```diff
@@ -33,21 +33,22 @@ program to a federated learning training program. The example will include:
 * SAG with NVFLARE Executor APIs
 
 
-## Tabular HIGGs dataset  
+## Binary classification with tabular data using HIGGS dataset  
 
-### HIGGS
+### HIGGS Dataset
+
+[HIGGS dataset](https://archive.ics.uci.edu/dataset/280/higgs) contains 11 million instances, each with 28 attributes, for binary classification to predict whether an event corresponds to the decayment of a Higgs boson or not.
 
-HIGGS dataset. This dataset contains 11 million instances, each with 28 attributes.
 The first 21 features (columns 2-22) are kinematic properties measured by the particle detectors in the accelerator. 
-The last seven features are functions of the first 21 features; these are high-level features derived by physicists to help discriminate between the two classes.
+The data has been produced using Monte Carlo simulations. The first 21 features are kinematic properties measured by the particle detectors in the accelerator. The last 7 features are functions of the first 21 features; these are high-level features derived by physicists to help discriminate between the two classes.
 
 Please note that the [UCI's website](https://archive.ics.uci.edu/dataset/280/higgs) may experience occasional downtime.
 
 With the HIGGs Dataset, we like to demonstrate traditional machine learning techniques in federated learning.
 These include:
 
 * Federated Statistics for tabular data
-* Federated Linear and Logistics Regression
+* Federated Logistic Regression
 * Federated Kmeans
-* Federated SVM with non-learner kernel
-* Federated (Horizontal) XGBoost
+* Federated SVM
+* Federated Horizontal XGBoost
```

Comment:
this part is duplicated with "examples/hello-world/step-by-step/higgs/README.md"
should we just add a link here?

Reply:
same for cifar-10, shall we do the same?

Reply:
sounds good